public/class-pw-admin-public.php

```
@@ -0,0 +1,1689 @@
+            }
+        }
+    }
+
+    /**
+     * Add custom cart column data after cart item name
+     * Hooks into: woocommerce_after_cart_item_name
+     * @since    1.0.0
+     * @param array $cart_item Cart item data
+     * @param string $cart_item_key Cart item key
+     */
+    public function add_custom_cart_column_data_revised($cart_item, $cart_item_key)
```

PHPMD (2.15.0)

Avoid unused parameters such as '$cart_item_key'. (Unused Code Rules)

(UnusedFormalParameter)